kroxylicious/src/test/java/io/kroxylicious/proxy/config/micrometer/MicrometerConfigurationTest.java

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright Kroxylicious Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.kroxylicious.proxy.config.micrometer;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+
+import org.junit.jupiter.api.Test;
+
+import io.micrometer.core.instrument.MeterRegistry;
+import io.micrometer.core.instrument.Tag;
+import io.micrometer.core.instrument.binder.MeterBinder;
+import io.micrometer.core.instrument.binder.jvm.JvmGcMetrics;
+import io.micrometer.core.instrument.binder.system.UptimeMetrics;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class MicrometerConfigurationTest {
```

Comment:
I think the tests would read better using `org.assertj.core.api.Assertions.assertThat`
e.g.

```java
    @Test
     public void testLocatesJvmBinder() {
         MicrometerConfiguration config = new MicrometerConfiguration(List.of("JvmGcMetrics"), null, null);
         List<Class<? extends MeterBinder>> binders = config.getBinders();
         assertThat(binders).contains(JvmGcMetrics.class);
     }
```

Reply:
switched to assertj